en/identity-server/6.1.0/docs/guides/login/oidc-logout-url-redirection.md

```diff
@@ -154,11 +154,28 @@ For a description of the parameters included in the HTML form, see [logout reque
 <td>Yes</td>
 </tr>
 <tr class="even">
+<td><code>               client_id              </code></td>
```

Comment:
Do we need all these spaces?

Reply:
The existing parameter names had similar spacing [1][2][3][4][5], so the same was followed here as well.

[1] https://github.com/wso2/docs-is/blob/master/en/identity-server/6.1.0/docs/guides/login/oidc-logout-url-redirection.md?plain=1#L168
[2] https://github.com/wso2/docs-is/blob/master/en/identity-server/6.1.0/docs/guides/login/oidc-logout-url-redirection.md?plain=1#L190
[3] https://github.com/wso2/docs-is/blob/master/en/identity-server/6.1.0/docs/guides/login/oidc-logout-url-redirection.md?plain=1#L195
[4] https://github.com/wso2/docs-is/blob/master/en/identity-server/6.1.0/docs/guides/login/oidc-logout-url-redirection.md?plain=1#L206
[5] https://github.com/wso2/docs-is/blob/master/en/identity-server/6.1.0/docs/guides/login/oidc-logout-url-redirection.md?plain=1#L211